docs/csharp/tour-of-csharp/tutorials/hello-world.yml

```
@@ -62,7 +62,7 @@ items:
     of the `aFriend` variable and its initial assignment.
```

Let's add a sentence here:

```suggestion
    of the `aFriend` variable and its initial assignment.
    
    > [!IMPORTANT]
    > Don't delete the declaration of `aFriend`. Add the code below following the existing declaration.
```